File: iOS/Swift/SampleSwift/AnchorVisual.swift
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.

class AnchorVisual {
    init() {
        node = nil
        identifier = ""
        cloudAnchor = nil
        localAnchor = nil
    }
    
    var node : SCNNode? = nil
    var identifier : String
    var cloudAnchor : ASACloudSpatialAnchor? = nil
    var localAnchor : ARAnchor? = nil
}

File: Xamarin/SampleXamarin.iOS/AnchorVisual.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.

using ARKit;
using Microsoft.Azure.SpatialAnchors;
using SceneKit;

namespace SampleXamarin.iOS
{
    public class AnchorVisual
    {
        public SCNNode node { get; set; }
        public string identifier { get; set; }
        public CloudSpatialAnchor cloudAnchor { get; set; }
        public ARAnchor localAnchor { get; set; }
    }
}